config/plugins.yaml

```diff
@@ -128,6 +128,9 @@ repo_milestone:
     maintainers_id: 3832091
     maintainers_team: pdig-maintainers
     maintainers_friendly_name: maintainers of falcosecurity/pdig
+  falcosecurity/falcoctl:
+    maintainers_team: falcoctl-maintainers
```

Comment:
Note: [`maintainerd_id`](https://github.com/kubernetes/test-infra/blob/master/prow/plugins/config.go#L478) is deprecated.